chat-plugins/scavengers.js

```diff
@@ -783,8 +783,23 @@ let commands = {
 		if (!room.game || !room.game.scavGame) return this.errorReply(`There is no scavenger game currently running.`);
 
 		let game = room.game.childGame || room.game;
-		let elapsed = Date.now() - game.startTime;
-		this.sendReplyBox(`The current ${game.gameType ? `<em>${game.gameType}</em> ` : ""}scavenger hunt by <em>${Chat.escapeHTML(Chat.toListString(game.hosts.map(h => h.name)))}${(game.hosts.some(h => h.userid === game.staffHostId) ? '' : ` (started by - ${Chat.escapeHTML(game.staffHostName)})`)}</em> has been up for: ${Chat.toDurationString(elapsed, {hhmmss: true})}<br />Completed (${game.completed.length}): ${game.completed.map(u => Chat.escapeHTML(u.name)).join(', ')}<br />`);
+		if (game.hosts.some(h => h.userid === user.userid) || game.staffHostId === user.userid) {
+			let str = `<div class="ladder" style="overflow-y: scroll; max-height: 300px;"><table style="width: 100%"><th><b>Question</b></th><th><b>Users on this Question</b></th>`;
+			for (let i = 0; i < game.questions.length; i++) {
+				let questionNum = i + 1;
+				let players = Object.values(game.players).filter(player => player.currentQuestion === i);
+				if (players.length === 0) {
+					str += `<tr><td>${questionNum}</td><td>None</td>`;
+				} else {
+					str += Chat.html`<tr><td>${questionNum}</td><td>${players.map(pl => pl.name).join(", ")}`;
+				}
+			}
+			str += Chat.html`<tr><td>Completed</td><td>${game.completed.length ? game.completed.map(pl => pl.name).join(", ") : 'None'}`;
+			this.sendReply('|raw|' + str + "</table></div>");
+		} else {
+			const elapsed = Date.now() - game.startTime;
+			this.sendReplyBox(Chat.html`The current <em>${game.gameType ? `${game.gameType} ` : ""}</em>scavenger hunt by <em>${Chat.toListString(game.hosts.map(h => h.name))}${(game.hosts.some(h => h.userid === game.staffHostId) ? '' : ` (started by - ${game.staffHostName})`)}</em> has been up for: ${Chat.toDurationString(elapsed, {hhmmss: true})}<br />Completed (${game.completed.length}): ${game.completed.map(u => u.name).join(', ')}<br />`);
```

Comment:
Also I am very against nested template strings in general.

Reply:
Also don't send empty `<em>` blocks just because you don't know how to write code that doesn't send them.